### ISSUE TITLE
Bug in service module with systemd

### ISSUE THREAD

**Issue by robparrott** [ansible/ansible#2449]
I'm seeing an error the first time I try to start a service on Fedora 17/18 using ansible. I believe this is related to systemd, because a google of the issue reveals the error comes from that codebase.

A stanza such as:

```
- name: start gluster service
  action: service name=glusterd state=started
```

results in the error:

```
{"failed": true, "item": ""} 
msg: Failed to add path to set.
```

The first retry is always successful. Really unsure as to the root cause, but this is kinda annoying!

**Comment by robparrott** [comment]
Some further poking reveals that this bug triggered on Fedora 17 by trying to restart the iptables service.

Turning on the syslog feature of the module, aI see in /var/log/messages the following line:

```
ansible-service: Command /bin/systemctl stop iptables iptables , daemonize True
```

Looks like the 2nd "iptables" argument is problematic.

**Comment by robparrott** [comment]
Bug looks like it's in LinuxService.service_control(), in particular how self.arguments attribute is built for systemctl.

**Comment by robparrott** [comment]
Fix here:

   https://github.com/ansible/ansible/pull/2460

**Comment by sfromm** [comment]
I proposed an alternate fix in PR #2466.

**Comment by mpdehaan** [comment]
We'll handle this via sfromm's PR, so closing this ticket since we have a ticket for the other already.

Thanks!
